src/components/LoadedTableWithCustomColumns.res

```diff
@@ -91,11 +91,11 @@ let make = (
       <Portal to={`${title}CustomizeColumn`}>
         <Button
           leftIcon=Button.CustomIcon(<Icon name="edit" size=16 />)
-          text="Customize Columns"
+          text=""
```

Comment:
Empty Text button?if empty then remove else see what can be done it's looking odd